integration-tests/config.ts

```diff
@@ -172,6 +182,8 @@ if (process.env['RUN_WITH_SECRET_KEY']) {
   providers.push(oxfordnetSecretKey);
 } else if (process.env['RUN_GHOSTNET_WITH_SECRET_KEY']) {
   providers.push(ghostnetSecretKey);
+} else if(process.env['RUN_NAIROBINET_WITH_SECRET_KEY']) {
```

Comment:
perhaps RUN_NAIROBIBOX_WITH_SECRET_KEY ?

Reply:
nairobinet_secret_key implies a Nairobi network running using a secret key (which in this case implies the usage of a sandbox).

either we keep the name, or just 'RUN_NAIROBIBOX'. Which one do you like better?